src/test/resources/serenity.conf

```diff
@@ -9,8 +9,12 @@ webdriver {
 }
 headless.mode = true
 
+drivers {
+    webdriver.chrome.driver = src/test/resources/drivers/chromedriver.exe
```

Comment:
please remove driver configuration due to webdriver.autodownload=true (WebDriverManager provides the binary file).
Also please remove driver package with *.exe